### ISSUE TITLE
Display full name in header if no first name given

### ISSUE THREAD

**Issue by RichardTaylor** [mysociety/alaveteli#4685]
Alaveteli currently shows users' first names in the navigation menu when they are logged in. 

If a user has a user name such as "J Bloggs" the navigation menu contains an item labelled "J". 

Perhaps if the first name is a single letter, then show the whole user name?

I imagine there are other circumstances where truncation to a "first name" wouldn't be appropriate too, such as an account in the name of an organisation. 

Really minor issue.

**Comment by RichardTaylor** [comment]
This also affects names of the form:

>Mrs Surname

Where "Mrs" appears in the navigation.

**Comment by FOIMonkey** [comment]
This is also an issue if a user uses a title such as Mr/Mrs/Lord/Duke/Sister etc 
![2022-07-15 (3)](https://user-images.githubusercontent.com/11962965/179235791-41e14aa5-a9fe-40e1-9a59-d9d02b509b76.png)

**Comment by WilliamWDTK** [comment]
We've had some correspondence with a user about this.

The user (A) had been looking at another user's (B) request. (A) noticed that (B)'s request had been refused due to not providing a full name, and A wondered whether this was because we'd abbreviated (B)'s name somehow.  Then, when I explained that what we send is the name we're given, (A) mentioned this issue of the name in the header. I am under the impression that the only name processing we do is for this account menu referenced in this issue.

(A) also referenced different naming habits in different cultures. Even in nearby countries like France, it's not uncommon for names to be written differently: `SMITH John`, rather than `John Smith`. So, I might be inclined to suggest disabling any name modifications at all.

**Comment by confirmordeny** [comment]
It may well be better to leave names unchanged. You can make FOI requests under the name of a company or if you are a sole trader, under a trading name.

**Comment by HelenWDTK** [comment]
This issue has been automatically closed due to a lack of discussion or resolution for over 12 months.
Should we decide to revisit this issue in the future, it can be reopened.
